android/src/main/java/com/reactnativecommunity/netinfo/NetInfoModuleImpl.java

```diff
@@ -63,17 +54,13 @@ public void onAmazonFireDeviceConnectivityChanged(boolean isConnected) {
         mConnectivityReceiver.setIsInternetReachableOverride(isConnected);
     }
 
-    @ReactMethod
+
     public void addListener(String eventName) {
-        numberOfListeners++;
-        mConnectivityReceiver.hasListener = true;
+        // iOS only
     }
 
-    @ReactMethod
-    public void removeListeners(Integer count) {
-        numberOfListeners -= count;
-        if (numberOfListeners == 0) {
-            mConnectivityReceiver.hasListener = false;
-        }
+
+    public void removeListeners(double count) {
+        // iOS only
```

Comment:
again, not sure about how this intersects with need to defer access of MAC address APIs

Reply:
yeah, same reason as above.

Reply:
just collapsing this one since it's really one issue and we have good discussion above